cloudferrylib/os/actions/check_networks.py

```diff
@@ -87,8 +86,90 @@ def run(self, **kwargs):
             LOG.critical('Network overlapping list:\n%s', overlapped_resources)
             raise exception.AbortMigrationError(
                 "There is a number of overlapping Network resources, so "
-                "migration process can not be continued. Resolve it please and"
+                "migration process can not "
+                "be continued. Resolve it please and"
                 " try again.")
+        self.check_conflicts_ports(src_net,
+                                   dst_net)
+
+    def check_conflicts_ports(self, src_net, dst_net):
+        LOG.info("Detecting ports conflicts...")
+        src_ports = self.get_ports('compute:', src_net)
+        dst_ports = self.get_ports('compute:', dst_net)
+        src_ports_by_fixip = self.ports_by_fixed_ips(src_ports)
+        dst_ports_by_fixip = self.ports_by_fixed_ips(dst_ports)
+        src_ports_by_mac = self.ports_by_mac(src_ports)
+        dst_ports_by_mac = self.ports_by_mac(dst_ports)
+        self.check_intersects_ports_fixips(src_ports_by_fixip,
+                                           dst_ports_by_fixip)
+        self.check_intersects_ports_mac(src_ports_by_mac,
+                                        dst_ports_by_mac)
```

Comment:
This will give false positive results when migration is run multiple times, the logic needs to be more sophisticated to handle the same (previously migrated) ports.